Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -530,6 +642,58 @@ import BraintreeDataCollector
         }
     }
 
+    private func performEditSwitchRequest(
+        editURL: URL,
```

Comment:
nit just to clean up the naming a bit:
```suggestion
        with url: URL,
```

Reply:
I just followed convention we had with performSwitchRequest.
I wanted to clarify what the url is for also.

Reply:
I think it's pretty clear since the method is called `performEditSwitchRequest`. Also the other one isn't particularly accurate since it's called `appSwitchURL` and is opening a web URL. What do you think?

Reply:
I don't have strong preference.